src/components/pages/doc.yml

```diff
@@ -11,5 +11,20 @@ commentsPanelTitle:
   en: You are viewing the document page in "comments" mode
   de: Du befindest dich im Kommentar-Modus
 commentsHeader:
-  en: Comment topics
-  de: Kommentar-Themen
+  en: Comments
+  de: Kommentare
+viewHistoryRestrictionTooltip_annonymousUser:
+  en: Viewing document history is only allowed for logged in users
+  de: Ansehen von vorherige Versionen ?? is nur für angemeldet Benutzern elaubt
+editRestrictionTooltip_annonymousUser:
+  en: Document editing is only allowed for logged in users
+  de: Bearbeitung von Dokumenten is nur für angemeldet Benutzern elaubt
```

Comment:
Maybe also here:

Log in in order to edit this document
Melde dich an, um dieses Dokument zu bearbeiten

Reply:
Much better, changed!